pkg/etcd/init/init.go

```diff
@@ -0,0 +1,126 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package init
+
+import (
+	"context"
+	"fmt"
+
+	clientv3 "go.etcd.io/etcd/client/v3"
+
+	"github.com/cilium/cilium/pkg/kvstore"
+)
+
+const (
+	rootRoleName = "root"
+)
+
+func InitEtcd(ctx context.Context, client *clientv3.Client, clusterName string) error {
```

Comment:
Nit. I'd add a comment stating that these operations are not idempotent, but you rely on the fact that this is a brand new etcd instance.

Reply:
I can change the function name to embed that meaning? `InitBlankEtcd` maybe? `ClustermeshEtcdFirstTimeSetup`?

Reply:
Makes sense to me, especially to mention clustermesh. What about `ClusterMeshEtcdInit`, or `InitEtcdForClusterMesh`? And I'd still mention that in the comment though, just to avoid getting confused in the future.

Reply:
Done.